lib/logic.rb

```diff
@@ -0,0 +1,106 @@
+class Game
+  attr_accessor :real_grid
+  def initialize()
+    @real_grid = Array.new(3, 0) { Array.new(3, 0) }
+    p @real_grid[0][1]
```

Comment:
This is a user-interface related logic and Game class shouldn't be responsible for it

Reply:
Two things:

1. Previously I mentioned that lines 5-12 (if i wasn't clear which lines, now I'm specifying) don't belong to your game logic. It's outputting something to the user interface (the terminal). Therefore, it belongs to the UI related code/file.
2. This file contains your Game class alone. So, I should say it must be named game.rb rather than plain logic.rb. While it may not appear to be a big deal, let's remember that we're being trained to work collaboratively with other engineers who may (in most projects) extend your project. Having a game.rb filename will make it intuitive for them and make it easier for other to collaborate and maintain your code.